README.md

```diff
@@ -107,7 +110,7 @@ heightInCentimeters = 180;
 
 We can try these out in REPL:
 
```

Comment:
I think we need to walk them through this: enter the first line, see that the return value is 50, enter the second, see that the return value is 180, then just run `heightInCentimeters` to see that the latest value is saved

Reply:
How does this look?

Reply:
Perfect!